### ISSUE TITLE
Accessibility Bug: There is no text alternative in the form of tooltip provided for the ‘Close’ and ‘Collapse’ button inside the dialog.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3119]
Fundamental core component: "Dailog"

Description: There is no text alternative in the form of tooltip provided for the ‘Close’ and ‘Collapse’ button inside the dialog.

Expected: The text alternative should be provided for every non-text content element which fulfills the purpose of the content.

Attached screenshot:

![image](https://user-images.githubusercontent.com/32538291/91555408-95502580-e94e-11ea-8d39-acfa338ef265.png)
